docs/rules.md

```diff
@@ -56,13 +56,14 @@ Please refer to [api-version-parameter-required.md](./api-version-parameter-requ
 
 ### APIVersionPattern
 
-The API Version parameter MUST be in the Year-Month-Date format (i.e. 2016-07-04.)  NOTE that this is the en-US ordering of month and date.
+The API Version parameter MUST be in the Year-Month-Date format (i.e. 2016-07-04.) NOTE that this is the en-US ordering of month and date.
 The date MAY optionally be followed by one of:
-* -preview - Indicates the API version is in (public) preview
-* -alpha
-* -beta
-* -rc (release candidate)
-* -privatepreview
+
+- -preview - Indicates the API version is in (public) preview
+- -alpha
+- -beta
+- -rc (release candidate)
+- -privatepreview
 
```

Comment:
this seems unrelated

Reply:
yes, need to run `rush regen-ruleindex` to fix it

Reply:
You might need to properly merge main to your branch

Reply:
I noticed that in Brett's PR too https://github.com/Azure/azure-openapi-validator/pull/424/files#diff-87f3af8240ee46430dbebe4e346fe9828cb1442f5175e470289fdf64183edc67
So, I thought that was something related to the project

Reply:
The changes apart from mine are from "rush regen-ruleindex"